share/scripts/inverse/run_gromacs_multidir.sh

```diff
@@ -0,0 +1,380 @@
+
+#checkpoint="state.cpt"
+checkpoint="$(csg_get_property cg.inverse.gromacs.mdrun.checkpoint)"
+
+#tpr="topol.tpr"
+tpr="$(csg_get_property cg.inverse.gromacs.topol)"
+
+#mdrun_opts="$(csg_get_property --allow-empty cg.inverse.gromacs.mdrun.opts)"
+grompp_opts="$(csg_get_property --allow-empty cg.inverse.gromacs.grompp.opts)"
+
+#grompp="gmx grompp"
+grompp="$(csg_get_property cg.inverse.gromacs.grompp.bin)"
+[[ -n "$(type -p ${grompp})" ]] && echo "using grompp binary '${grompp}'" || die "${0##*/}: grompp binary '${grompp}' not found (override by cg.inverse.gromacs.grompp.bin)"
+
+if is_done "Simtasks"; then
+
+    echo -e "\nAll Simulation tasks done - skipping\n"
+
+else
+
+    for ((it=0;it<mdirs;it++)); do
+	this_dir="sim_${it}"
+
+	if [[ -d $this_dir ]]; then
+
+	    cd $this_dir || die "${0##*/}: cd $this_dir failed"
+
+	    if is_done "Simulation"; then 
+		echo -e "\nSimulation task ${it} done - skipping\n"
+		cd ../
+		continue; 
+	    fi
+
+	    cd ../
+
+	else
+	    echo -e "\nSimulation task $it created at $(date)\n"
+	    mkdir -p $this_dir || die "${0##*/}: mkdir -p $this_dir failed (unfinished simulation exists?)"
+	fi
+
+	cd $this_dir || die "${0##*/}: cd $this_dir failed"
+
+	filelist="$(csg_get_property --allow-empty cg.inverse.filelist)"
+
+	if is_done "Filecopy"; then
+        #check for needed files 
+	    echo "Filecopy already done"
+	    for f in $filelist; do
+
+		[[ -f $f ]] || cp_from_main_dir "$f"
+
+		echo Comparing "$(get_main_dir)/$f" "$f"
+		[[ -z $(type -p cmp) ]] && echo "${0##*/}: program 'cmp' not found, comparision skipped" && continue
+
+		cmp "$(get_main_dir)/$f" "$f" && echo "Unchanged" || \
+		msg --color blue --to-stderr "${0##*/}: WARNING: file '$f' in the main dir was changed since the last execution, this will have no effect on the current iteration, to take effect remove the current iteration ('${this_dir##*/}')"
+
+	    done
+	else
+        #get needed files 
+	    echo "Copying the needed files"
+	    [[ -n ${filelist} ]] && cp_from_main_dir "$filelist"
+
+	    mark_done "Filecopy"
+	fi
+
+	if [[ $1 != "--pre" ]]; then
+        #in a presimulation usually do care about traj and temperature
+	   check_temp || die "${0##*/}: check of tempertures failed"
+	   if [[ ${traj} == *.xtc ]]; then
+	      [[ $(get_simulation_setting nstxtcout 0) -eq 0 ]] && die "${0##*/}: trajectory type (cg.inverse.gromacs.traj) is '${traj##*.}', but nstxtcout is 0 in $mdp. Please check the setting again and remove the current step."
+	   elif [[ ${traj} == *.trr ]]; then
+		[[ $(get_simulation_setting nstxout 0) -eq 0 ]] && die "${0##*/}: trajectory type (cg.inverse.gromacs.traj) is '${traj##*.}', but nstxout is 0 in $mdp. Please check the setting again and remove the current step."
+	   else
+	       die "${0##*/}: error trajectory type '${traj##*.}' (ending from '${traj}') is not supported"
+	   fi
+	fi
+
+        [[ -f ${checkpoint} ]] && echo "using checkpoint file '${checkpoint}'"
+
```

Comment:
Meaning: you'd like to have a specific script for the currently repeated pieces? - in this case for the bits invoked by '-pre' option?